cactus/tests/test_basic.py

```diff
@@ -113,4 +114,30 @@ def test_current_page(self):
             self.assertEqual('True', f.read())
 
         with open(os.path.join(self.path, '.build', other), 'rU') as f:
-            self.assertEqual('False', f.read())
\ No newline at end of file
+            self.assertEqual('False', f.read())
+
+
+class TestPage(SiteTestCase):
```

Comment:
Would be ideal to break out the other tests for `Page` into that class itself then (or into a separate file altogether).